tests/test_model.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import pytest
+import peewee
+
+from tracboat.gitlab import model
+
+
+@pytest.mark.parametrize('version', [
+    '8.4',
+    '8.5',
+    '8.7',
+    '8.13',
+    '8.15',
+    '8.16',
+    '8.17',
+    '9.0.0'
```

Comment:
I agree, I was thinking it could be useful to make the separation between major and minor in model file name explicit, e.g. rename `model817.py` to `model8.17.py` or `model_8_17.py`, something like that, bare `817` is a bit ambiguous imho.